support/doc/production.md

```diff
@@ -104,11 +104,29 @@ server {
   listen [::]:443 ssl http2;
   server_name peertube.example.com;
 
-  # For example with Let's Encrypt
+  # For example with Let's Encrypt (you need a certificate to run https)
   ssl_certificate      /etc/letsencrypt/live/peertube.example.com/fullchain.pem;
   ssl_certificate_key  /etc/letsencrypt/live/peertube.example.com/privkey.pem;
-  ssl_trusted_certificate /etc/letsencrypt/live/peertube.example.com/chain.pem;
-
+  ssl_dhparam /etc/nginx/dhparam.pem; # openssl dhparam -out /etc/nginx/dhparam.pem 4096
+  
+  # Security hardening (as of 11/02/2018)
+  ssl_protocols TLSv1.3, TLSv1.2;# TLSv1.3 requires nginx >= 1.13.0 else use only TLSv1.2
+  ssl_prefer_server_ciphers on;
+  ssl_ciphers ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384:!3DES;
```

Comment:
Remove DHE since it's much slower than ECDHE and all modern clients use it.
Checkout https://mozilla.github.io/server-side-tls/ssl-config-generator/

Reply:
Indeed! I didn't think about removing it, which makes sense for elliptic curve computations.